File: frontend/admin/src/datas/businessTypes.ts
export interface IBusinessType {
  label: string;
  id: number;
}

export function getBusinessType(id: number) {
  return businessTypes[id - 1];
}

export const businessTypes = [
  {
    id: 1,
    label: "SERVICE",
  },
  {
    id: 2,
    label: "TRADING_COMPANY",
  },
  {
    id: 3,
    label: "MANUFACTURER",
  },
];

File: mobile/lib/components/business_types.dart
import 'package:flutter/material.dart';
import 'package:sdconnect_mobile/generated/l10n.dart';

const List<Map> kBusinessTypes = [
  {
    "id": 1,
    "label": "SERVICE",
  },
  {
    "id": 2,
    "label": "TRADING_COMPANY",
  },
  {
    "id": 3,
    "label": "MANUFACTURER",
  },
];

String getBusinessTypesString(BuildContext context, List<int> businessTypeIds) {
  List<String> businessTypesMap = [
    S.of(context).businessTypeService,
    S.of(context).businessTypeTrading,
    S.of(context).businessTypeManufacturer
  ];

  return businessTypeIds.map((id) => businessTypesMap[id - 1]).join(", ");
}
